views/layout.ejs

```diff
@@ -48,88 +48,22 @@
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
       <% if (sails.config.env === 'dev') {%>
-        console.log('GA tracking code: %c UA-67412132-1', ';color: #6695DC;')
         ga('create', 'UA-67412132-1', 'auto');
       <% } else { %>
-        console.log('GA tracking code: %c UA-67412132-3', ';color: #6695DC;')
         ga('create', 'UA-67412132-3', 'auto');
       <% } %>
+    </script>
 
-      ga('send', 'pageview');
-
-      /**
-       * sends a GA event, used to centralise the events and pages name
-       * @param {String} root
-       * @param {String} page
-       * @param {String} section
-       * @param {String} action
-       * @param {String} value
-       */
-      window.sendGAEvent = function sendGAEvent(root, page, section, action, value) {
-
-        // normalising the root name
-        if (/audience/ig.test(root)) root = 'audiences';
-        if (/project/ig.test(root)) root = 'projects';
-
-        console.info(
-          'tracking: %c' + root + ' ' + page +
-          (' ' + (section || '')) +
-          (' ' + (action || '')) +
-          (' ' + (value || '')), ';color: #6695DC;');
-
-				ga('send', 'event', root, page, section, action, value);
-
-        <%if (typeof req.session.user !== 'undefined') {%>
-          ga('send', 'event', '<%= req.session.user.email %>', root, page, section, action, value, new Date().toISOString());
-        <%}%>
-
-      };
-
-      <%if (req.session.user) {%>
-        sendGAEvent('user page request', '<%= req.session.user.email %>', window.location.href);
-      <%}%>
-
-      // Sentry configuration
       <% if (sails.config.env === 'dev') {%>
-        // dev
-        Raven.config('https://57ddbec2382c4d02b3fc420a82fc3fd8@app.getsentry.com/59627').install();
-      <% } else {%>
-        // prod
-        Raven.config('https://1aaa7d20cef449318c8a056e2382fad4@app.getsentry.com/59625').install()
-      <% } %>
-
-      <% if (req.session.user) { %>
-        // Set the user context for raven error logging
-        Raven.setUserContext({
-          email: '<%= req.session.user.email %>',
-          id: '<%= req.session.user.id %>'
-        });
+        <script>
+          // Sentry configuration
+          Raven.config('https://57ddbec2382c4d02b3fc420a82fc3fd8@app.getsentry.com/59627').install();
```

Comment:
Cool yeah I wasn't sure so I left it

On Fri 12 Feb 2016 at 18:26, Daniel Oliveira <notifications@github.com>
wrote:

> In views/layout.ejs
> <https://github.com/bxfsh/sails-hook-user/pull/4#discussion_r52778332>:
>
> > -        // dev
> > -        Raven.config('https://57ddbec2382c4d02b3fc420a82fc3fd8@app.getsentry.com/59627').install();
> > -      <% } else {%>
> > -        // prod
> > -        Raven.config('https://1aaa7d20cef449318c8a056e2382fad4@app.getsentry.com/59625').install()
> > -      <% } %>
> > -
> > -      <% if (req.session.user) { %>
> > -        // Set the user context for raven error logging
> > -        Raven.setUserContext({
> > -          email: '<%= req.session.user.email %>',
> > -          id: '<%= req.session.user.id %>'
> > -        });
> > +        <script>
> > +          // Sentry configuration
> > +          Raven.config('https://57ddbec2382c4d02b3fc420a82fc3fd8@app.getsentry.com/59627').install();
>
> I think we can remove this, this is for Sentry and we no longer use it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bxfsh/sails-hook-user/pull/4/files#r52778332>.
>